scripts/rest_client.py

```diff
@@ -379,7 +386,11 @@ def main(args):
         print_commands()
         return
 
-    print(getattr(client, command)(*args["command_arguments"]))
+    output = getattr(client, command)(*args["command_arguments"])
+    if args['json']:
+        print(json.dumps(output))
+    else:
+        print(output)
```

Comment:
hm, using json dumps is a good idea for this - but honestly, do we really need it optional?

having it in "python print" way was just lazyness, not something that's required for any functionality i'd think.

Reply:
So we can just print using json by defualt?